packages/cli/src/commands/composite/from-model.ts

```diff
@@ -0,0 +1,52 @@
+import { Composite } from '@glazed/devtools'
+import { Command, type CommandFlags, getArrayArg } from '../../command.js'
+import { Flags } from '@oclif/core'
+import fs from 'fs-extra'
+
+type Flags = CommandFlags & {
+  output?: string
+}
+
+export default class CompositeFromModel extends Command<Flags> {
+  static strict = false
+
+  static description = 'create a Composite from a list of model streams'
+
+  static flags = {
+    ...Command.flags,
+    output: Flags.string({
+      char: 'o',
+      description: 'path to the file where the composite representation should be saved',
+    }),
+  }
+
+  async run(): Promise<void> {
+    const modelStreamIDs = getArrayArg(this.argv, this.flags)
+
+    if (modelStreamIDs.length === 0) {
+      this.spinner.fail('Missing list of model streamIDs')
+      return
+    }
+
+    this.spinner.start('Creating the composite...')
+    try {
+      const composite = await Composite.fromModels({
+        ceramic: this.ceramic,
+        models: modelStreamIDs,
+      })
+      const encodedAsJSON = JSON.stringify(composite.toJSON(), null, 2)
+      if (this.flags.output !== undefined) {
+        const output = this.flags.output
+        await fs.writeFile(output, encodedAsJSON)
+        this.spinner.succeed(
+          `Composite was created and its encoded representation was saved in ${output}`
+        )
+      } else {
+        this.spinner.succeed(encodedAsJSON)
```

Comment:
Could you remove the display of the spinner here so the output can be directly piped please?

Reply:
Will do! 

The spinner is still gonna be used in the parent `Command` class when the `--key` flag is used, but I'm gonna  fix this either when working on the new `did:*` commands or on adding support for the standard I/O, if that makes sense.

Reply:
Can you check how OCLIF and Ora (the spinner library) handle TTY please? I think they already support options or even good defaults for piping commands, we mostly need to make sure we don't do anything that prevents this.

Reply:
Will do, thanks, @PaulLeCam :) !

Do you think it's best to work on this in this PR or can leave for the time when I focus on the work on standard I/O?

Reply:
Yes no problem doing this in a separate PR if you prefer, it will affect lots of other commands as well anyway so might be simpler to handle them all at once.